tools/xcodeproj_shims/xcodeproj-installer.sh

```diff
@@ -34,6 +34,12 @@ do
 done
 cp "$(installer_short_path)" "${installers_dir}/"
 
+build_wrapper_runfile_short_paths="$(build_wrapper_runfile_short_paths)"
+for BUILD_WRAPPER_PATH in $build_wrapper_runfile_short_paths
+do
+  cp -r "$BUILD_WRAPPER_PATH" "${stubs_dir}/"
```

Comment:
Mind clarifying why copying these files is necessary? Is it so `xcode_version_computation` is available during runtime to the build wrapper?

Reply:
Yep currently there is a bash script that has runfile dependencies:
https://github.com/bazel-ios/rules_ios/pull/274/files#diff-15b94d1cfaca23eeea9cfded406d994f36fc7349f5786e2746eedb20ebfd671cR42

Reply:
Maybe just port the entire thing to swift here shortly 🤔